pyasic/miners/backends/marathon.py

```diff
@@ -302,3 +307,33 @@ async def _get_wattage_limit(
                 return web_miner_config["mode"]["concorde"]["power-target"]
             except LookupError:
                 pass
+
+    async def _get_pools(self, web_pools: dict = None) -> List[PoolMetrics]:
+        if web_pools is None:
+            try:
+                web_pools = await self.web.pools()
+            except APIError:
+                pass
+
+        pools_data = []
+        if web_pools is not None:
+            try:
+                pools = web_pools.get("POOLS", [])
+                for pool_info in pools:
+                    url = pool_info.get("URL")
+                    pool_url = PoolUrl.from_str(url) if url else None
+                    pool_data = PoolMetrics(
+                        accepted=pool_info.get("Accepted"),
+                        rejected=pool_info.get("Rejected"),
+                        get_failures=pool_info.get("Get Failures"),
+                        remote_failures=pool_info.get("Remote Failures"),
+                        active=pool_info.get("Stratum Active"),
+                        alive=pool_info.get("Status") == "Alive",
+                        url=pool_url,
+                        user=pool_info.get("User"),
+                        index=pool_info.get("POOL"),
+                    )
```

Comment:
This is not correct, see the above comment.

Reply:
do you mean I should change from web to rpc?

Reply:
No, I mean that the parsing is incorrect, I sent an example result in that earlier comment.

Reply:
```
    [
        {
            "index": 0,
            "url": "stratum+tcp://ca.stratum.braiins.com:3333",
            "user": "BR7609.S19k",
            "status": "Alive",
            "priority": 0,
            "getworks": 4,
            "accepted": 52,
            "rejected": 0,
            "discarded": 0,
            "stale": 0,
            "diff": "135355",
            "diff1": 1824777,
            "diffa": 1824777,
            "diffr": 0,
            "diffs": 0,
            "lsdiff": 8192,
            "lstime": "2.189571873s",
            "latency": 83,
            "block_height": 858555,
            "submit_count": 52,
            "connect_time": "1m15.189620331s",
            "reconnect_count": 1,
            "extra_nonce_change_count": 0,
        },
        {
            "index": 1,
            "url": "stratum+tcp://us-east.stratum.braiins.com:3333",
            "user": "BR7609.S19k",
            "status": "Alive",
            "priority": 1,
            "getworks": 5,
            "accepted": 0,
            "rejected": 0,
            "discarded": 0,
            "stale": 0,
            "diff": "512",
            "diff1": 0,
            "diffa": 0,
            "diffr": 0,
            "diffs": 0,
            "lsdiff": 1024,
            "lstime": "479084h37m20.189642998s",
            "latency": 63,
            "block_height": 858555,
            "submit_count": 0,
            "connect_time": "1m11.189657956s",
            "reconnect_count": 1,
            "extra_nonce_change_count": 0,
        },
        {
            "index": 2,
            "url": "stratum+tcp://stratum.braiins.com:3333",
            "user": "BR7609.S19k",
            "status": "Alive",
            "priority": 2,
            "getworks": 4,
            "accepted": 0,
            "rejected": 0,
            "discarded": 0,
            "stale": 0,
            "diff": "1024",
            "diff1": 0,
            "diffa": 0,
            "diffr": 0,
            "diffs": 0,
            "lsdiff": 8192,
            "lstime": "479084h37m20.189672498s",
            "latency": 62,
            "block_height": 858555,
            "submit_count": 0,
            "connect_time": "1m11.189684706s",
            "reconnect_count": 1,
            "extra_nonce_change_count": 0,
        },
    ]
```